cmd/exporter.go

```diff
@@ -192,15 +202,14 @@ func Run(ctx *cli.Context) error {
 		log.Warn("TLS verification has been disabled. Readiness checks won't be operated.")
 	}
 
-	// Graceful shutdowns
-	done := make(chan os.Signal, 1)
-	signal.Notify(done, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
-
 	// Expose the registered metrics via HTTP
 	mux := http.NewServeMux()
 	mux.HandleFunc("/health/live", health.LiveEndpoint)
 	mux.HandleFunc("/health/ready", health.ReadyEndpoint)
-	mux.Handle("/metrics", promhttp.Handler())
+	mux.Handle("/metrics", promhttp.HandlerFor(metrics, promhttp.HandlerOpts{
+		Registry:          metrics,
+		EnableOpenMetrics: true,
```

Comment:
This is needed when preventing the use of `init()` function ( init should be used only when non-avoidable, like imports for side effects).
Do you mean to make parametric the `/metrics` endpoint?

Reply:
I was referring to `EnableOpenMetrics` being set to `true`

According to the doc:
> // If true, the experimental OpenMetrics encoding is added to the
    // possible options during content negotiation. Note that Prometheus
    // 2.5.0+ will negotiate OpenMetrics as first priority. OpenMetrics is
    // the only way to transmit exemplars. However, the move to OpenMetrics
    // is not completely transparent. Most notably, the values of "quantile"
    // labels of Summaries and "le" labels of Histograms are formatted with
    // a trailing ".0" if they would otherwise look like integer numbers
    // (which changes the identity of the resulting series on the Prometheus
    // server).

Reply:
Ohhh I see... I thought it was already the default, I misread sorry... yes that must come from a `Config` parameter